sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_constants.py

```diff
@@ -0,0 +1,76 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import logging
+import platform
+from os import environ
+from pathlib import Path
+
+from azure.monitor.opentelemetry.exporter._connection_string_parser import ( # pylint: disable=import-error
+    ConnectionStringParser,
+)
+
+# --------------------Configuration------------------------------------------
+
+CONNECTION_STRING_ARG = "connection_string"
+DISABLE_LOGGING_ARG = "disable_logging"
+DISABLE_METRICS_ARG = "disable_metrics"
+DISABLE_TRACING_ARG = "disable_tracing"
+LOGGING_EXPORT_INTERVAL_MS_ARG = "logging_export_interval_ms"
+SAMPLING_RATIO_ARG = "sampling_ratio"
+
+
+# --------------------Diagnostic/status logging------------------------------
+
+_LOG_PATH_LINUX = "/var/log/applicationinsights"
+_LOG_PATH_WINDOWS = "\\LogFiles\\ApplicationInsights"
+_IS_ON_APP_SERVICE = "WEBSITE_SITE_NAME" in environ
+# TODO: Add environment variable to enabled diagnostics off of App Service
+_IS_DIAGNOSTICS_ENABLED = _IS_ON_APP_SERVICE
+# TODO: Enabled when duplicate logging issue is solved
+# _EXPORTER_DIAGNOSTICS_ENABLED_ENV_VAR = (
+#     "AZURE_MONITOR_OPENTELEMETRY_DISTRO_ENABLE_EXPORTER_DIAGNOSTICS"
+# )
+logger = logging.getLogger(__name__)
+_CUSTOMER_IKEY = "unknown"
```

Comment:
instrumentation_key below can be None, so one remedy for the mypy error is:

```python
_CUSTOMER_IKEY: Optional[str] = "unknown"
```

Reply:
done